ui/appui-react/src/appui-react/statusfields/SelectionScope.tsx

```diff
@@ -33,47 +32,39 @@ interface SelectionScopeFieldProps extends CommonProps {
  * in the Redux state.
  * @public
  */
-class SelectionScopeFieldComponent extends React.Component<SelectionScopeFieldProps> {
-  private _label = UiFramework.translate("selectionScopeField.label");
-  private _toolTip = UiFramework.translate("selectionScopeField.toolTip");
-  private _scopeOptions: SelectOption<string>[] = [];
+function SelectionScopeFieldComponent(props: SelectionScopeFieldProps) {
+  const [label] = React.useState(UiFramework.translate("selectionScopeField.label"));
+  const [toolTip] = React.useState(UiFramework.translate("selectionScopeField.toolTip"));
```

Comment:
There is no reason to have `useState` here since these are strings, and the `translate` is always called but dropped (which means that if an application ever implement live language switch, the language switch will be missed)

Reply:
Changed for consistency.